src/_guides/libraries/c-interop.md

```diff
@@ -125,10 +125,18 @@ Once you understand the hello_world example, you should be ready to look at the
 [other dart:ffi examples](#examples).
 
 
+## Packaging and loading C libraries
+
+How you get C libraries, distribute them, and load them into your code
```

Comment:
"get C libraries" is a bit confusing here. I would just say "How you package C libraries, distribute them, and load them..."

Reply:
Those are the right steps (for 3 I've seen both the terms packaging and distributing used depending on the methods used.

Rephrasing my feedback in terms of these, my point was that the docs really don't cover 1. We're assuming that the developer already has some C code (wether source or compiled) and then are trying to do steps 2-4.